src/common/ceph_context.cc

```diff
@@ -530,9 +551,20 @@ void CephContext::do_command(std::string command, cmdmap_t& cmdmap,
 }
 
 
+static bool is_daemon(uint32_t module_type)
```

Comment:
Nit: shouldn't this just come from ```CODE_ENVIRONMENT_DAEMON```? There are lots of other daemons like rbd-mirror that should also probably get the daemon default overrides.